download_reads.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from importlib.machinery import SourceFileLoader
+
+run_name = str(sys.argv[1])
+run_type = str(sys.argv[2])
+
+def bs_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(1)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('|', '') for x in tmp]
+    tmp=[re.sub(r"\s+", ' ', x) for x in tmp]
+    return tmp
+
+# Slack_sdk values needed for sending messages to Slack. Uses an API already set up on the Slack website for the UPHL Workspace to post messages on the notifications channel.
+# Function makes sending Slack messages as easy as using the print funcition.
+
+config = SourceFileLoader("config","/Volumes/IDGenomics_NAS/Bioinformatics/jarnn/config.py").load_module()
+
+client = WebClient(token=config.token)
+channel_id = config.channel_id
+def slack_message(string):
+    try:
+
+        result = client.chat_postMessage(
+            channel=channel_id,
+            text=string
+        )
+
+    except:
+        print("Slack Error")
+
+
+# Make dir that reads and anaylsis files will be saved to
+
+if run_type == "granduer":
+    path = "/Volumes/IDGenomics_NAS/WGS_Serotyping/%s/Sequencing_reads/Raw"% run_name
+if run_type == "mycosnp":
+    path = "/Volumes/IDGenomics_NAS/fungal/%s/Sequencing_reads/Raw"% run_name
+
+try:
+    os.makedirs(path)
+except:
+    print("Directory Already Exits, Make sure Reads are not already downloaded and delete Directory %s" % datetime.now())
+    sys.exit()
+
+
+# Once the run is completed it will be downloaded
+bashCommand='bs list --config=bioinfo runs'
+tmp=bs_out(bashCommand)
+tmp=pd.DataFrame(index=[row.split()[2] for row in tmp[1:]], columns=tmp[0].split()[1:], data=[row.split()[1:4] for row in tmp[1:]])
+idd=tmp.at[run_name,'Id']
+
+
+# First the samplesheet must be downloaded
+bashCommand = """bs run download --config=bioinfo
+          --output= "%s"
+          --id=%s
+          --extension=csv
+""" % (path,idd)
```

Comment:
I'm not seeing the Sample sheet getting downloaded. I think it's because of the space on line 73.
`--output= "%s"`  should be `--output="%s"`